app/assets/javascripts/components/client/ClientList.jsx

```diff
@@ -0,0 +1,19 @@
+class ClientList extends React.Component {
```

Comment:
Not sure if this file is being used anywhere else in the code, is it necessary? Eric already made one for the all clients page